### ISSUE TITLE
core-jaeger crashes and restart infinetly

### ISSUE THREAD

**Issue by rkamath3** [kyma-project/kyma#1525]
**Description**
While installing the kyma v.0.4.3, on kubernetes cluster (openStack) the core-jaeger fails to start.  I am using an internal DNS server and a certificate issued by an internal CA to my domain. The certificate is issued to my domain, let us consider that as `*.test.sap`. 

kube-proxy logs
`1.5414072750216842e+09    info    keycloak-proxy/server.go:682    attempting to retrieve configuration discovery url    {"url": "https://dex.test.sap", "timeout": "30s"}
1.541407275023699e+09    warn    keycloak-proxy/server.go:688    failed to get provider configuration from discovery    {"error": "Get https://dex.test.sap/.well-known/openid-configuration: EOF"}`  

kyma version 0.4.3
``` 
kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T20:17:28Z", GoVersion:"go1.10.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.7", GitCommit:"0c38c362511b20a098d7cd855f1314dad92c2780", GitTreeState:"clean", BuildDate:"2018-08-20T09:56:31Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
````
**Expected result**
Kyma installation is sucessful and kyma console is accessible through a browser

**Actual result**
Error in installation and the console login fails with `There was an error while requesting a resource:
Unauthorized` error.

**Steps to reproduce**
- Install kyma on OpenStack Kubernetes cluster

**Troubleshooting**
Re installation - failed
```
Kyma-iinstallation log
Checking state of kyma installation...hold on
Status: InProgress, description: Processing component core
apiVersion: installer.kyma-project.io/v1alpha1
kind: Installation
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"installer.kyma-project.io/v1alpha1","kind":"Installation","metadata":{"annotations":{},"labels":{"action":"install"},"name":"kyma-installation","namespace":"default"},"spec":{"components":[{"name":"cluster-essentials","namespace":"kyma-system"},{"name":"istio","namespace":"istio-system"},{"name":"prometheus-operator","namespace":"kyma-system"},{"name":"provision-bundles"},{"name":"dex","namespace":"kyma-system"},{"name":"core","namespace":"kyma-system"}],"url":"https://github.com/kyma-project/kyma/archive/0.4.3.tar.gz","version":"0.0.1"}}
  clusterName: ""
  creationTimestamp: 2018-11-04T20:35:44Z
  generation: 1
  labels:
    action: install
  name: kyma-installation
  namespace: default
  resourceVersion: "1232332"
  selfLink: /apis/installer.kyma-project.io/v1alpha1/namespaces/default/installations/kyma-installation
  uid: 340cb206-e071-11e8-a411-0a58c0a8003c
spec:
  components:
  - name: cluster-essentials
    namespace: kyma-system
    release: ""
  - name: istio
    namespace: istio-system
    release: ""
  - name: prometheus-operator
    namespace: kyma-system
    release: ""
  - name: provision-bundles
    namespace: ""
    release: ""
  - name: dex
    namespace: kyma-system
    release: ""
  - name: core
    namespace: kyma-system
    release: ""
  url: https://github.com/kyma-project/kyma/archive/0.4.3.tar.gz
  version: 0.0.1
status:
  conditions:
  - lastProbeTime: 2018-11-04T20:35:44Z
    lastTransitionTime: 2018-11-04T20:35:44Z
    status: "False"
    type: Installed
  - lastProbeTime: 2018-11-04T20:35:44Z
    lastTransitionTime: 2018-11-04T20:35:44Z
    status: "True"
    type: Installing
  - lastProbeTime: 2018-11-04T20:35:44Z
    lastTransitionTime: 2018-11-04T20:35:44Z
    status: "True"
    type: InProgress
  - lastProbeTime: 2018-11-04T20:35:44Z
    lastTransitionTime: 2018-11-04T20:35:44Z
    status: "False"
    type: Error
  description: Processing component core
  state: InProgress
  url: ""
  version: ""
----------
NAMESPACE          NAME                                                     READY     STATUS             RESTARTS   AGE
istio-system       istio-citadel-6bbfbbbc59-9gd9q                           1/1       Running            0          17h
istio-system       istio-egressgateway-7f7cbdd6bb-42rcz                     1/1       Running            0          17h
istio-system       istio-galley-784dd6d956-7rj2l                            1/1       Running            0          17h
istio-system       istio-ingressgateway-85758b8fb-hfgbt                     1/1       Running            0          17h
istio-system       istio-pilot-7888b94bf8-6svgl                             2/2       Running            0          17h
istio-system       istio-policy-97b7fd6dc-dgbnf                             2/2       Running            0          17h
istio-system       istio-sidecar-injector-7799fc464-qs8p6                   1/1       Running            0          17h
istio-system       istio-statsd-prom-bridge-6495856cff-9xr2w                1/1       Running            0          17h
istio-system       istio-telemetry-59d877d76d-kg6m9                         2/2       Running            0          17h
istio-system       istio-webhook-657b9c8cc8-plnnn                           1/1       Running            0          17h
kube-system        kube-dns-7dc9f9f4c5-jgsr5                                3/3       Running            0          18d
kube-system        kubernetes-dashboard-7b9c7bc8c9-5p2ck                    1/1       Running            0          18d
kube-system        tiller-deploy-55c5bf67fd-f75gx                           1/1       Running            0          18d
kyma-installer     kyma-installer-64d648c866-rpvtp                          1/1       Running            0          17h
kyma-integration   connector-service-67c87cdbfd-fxwn2                       2/2       Running            0          17h
kyma-integration   metadata-service-5464458cfc-9xx8h                        2/2       Running            0          17h
kyma-integration   remote-environment-controller-0                          1/1       Running            0          17h
kyma-system        alertmanager-core-0                                      2/2       Running            0          17h
kyma-system        api-controller-78bdddd988-spjsj                          1/1       Running            0          17h
kyma-system        apiserver-proxy-6b8764d679-55xqb                         2/2       Running            0          17h
kyma-system        apiserver-proxy-6b8764d679-8fmxz                         2/2       Running            0          17h
kyma-system        cluster-essentials-pod-preset-webhook-66dfcfc69b-mb8hj   1/1       Running            0          17h
kyma-system        configurations-generator-7f455648b6-knhn2                2/2       Running            0          17h
kyma-system        core-binding-usage-controller-6dbd596f89-5vqrl           2/2       Running            0          17h
kyma-system        core-catalog-apiserver-84f68f5b47-cfplf                  1/1       Running            4          17h
kyma-system        core-catalog-controller-manager-6b9b8bf4d9-snxv8         1/1       Running            5          17h
kyma-system        core-catalog-etcd-stateful-0                             1/1       Running            1          17h
kyma-system        core-catalog-etcd-stateful-1                             1/1       Running            0          17h
kyma-system        core-catalog-etcd-stateful-2                             1/1       Running            0          17h
kyma-system        core-catalog-ui-864ddbc697-x9wc7                         1/1       Running            0          17h
kyma-system        core-console-8698d747bb-xw48b                            1/1       Running            0          17h
kyma-system        core-content-ui-847c964686-prcvj                         1/1       Running            0          17h
kyma-system        core-environments-6d6f778854-rgkbw                       1/1       Running            0          17h
kyma-system        core-exporter-kube-state-74cb9677d6-qrtcr                2/2       Running            0          17h
kyma-system        core-exporter-node-c7v49                                 1/1       Running            0          17h
kyma-system        core-exporter-node-d7gxv                                 1/1       Running            0          17h
kyma-system        core-exporter-node-jt889                                 1/1       Running            0          17h
kyma-system        core-grafana-0                                           2/2       Running            0          17h
kyma-system        core-helm-broker-64b946d5bf-7xfvq                        1/1       Running            0          17h
kyma-system        core-helm-broker-etcd-stateful-0                         1/1       Running            0          17h
kyma-system        core-helm-broker-reposerver-6dbb4948b6-2mqs8             1/1       Running            0          17h
kyma-system        core-instances-ui-759c95bd96-xf8h4                       1/1       Running            0          17h
kyma-system        core-jaeger-84f96b5d6b-cdvst                             2/3       CrashLoopBackOff   9          31m
kyma-system        core-kubeless-controller-manager-6c5b7cf5b4-57tvz        1/1       Running            0          17h
kyma-system        core-lambdas-ui-f4d57c67d-nzxqw                          1/1       Running            0          17h
kyma-system        core-logging-logspout-94vxs                              1/1       Running            3          17h
kyma-system        core-logging-logspout-nhx8v                              1/1       Running            3          17h
kyma-system        core-logging-logspout-qvzrs                              1/1       Running            3          17h
kyma-system        core-logging-oklog-0                                     1/1       Running            0          17h
kyma-system        core-minio-5cd8bf6868-s8q49                              1/1       Running            0          17h
kyma-system        core-nats-streaming-0                                    2/2       Running            0          17h
kyma-system        core-nginx-ingress-controller-7fc9cd87fb-b4nx9           1/1       Running            0          17h
kyma-system        core-nginx-ingress-default-backend-ffbdb65c-cl92z        1/1       Running            0          17h
kyma-system        core-publish-5bb5cfbdc6-b7scn                            2/2       Running            3          17h
kyma-system        core-publish-5bb5cfbdc6-ddxcs                            2/2       Running            3          17h
kyma-system        core-push-69489fdfc7-6k6cv                               2/2       Running            3          17h
kyma-system        core-push-69489fdfc7-rxbvs                               2/2       Running            3          17h
kyma-system        core-remote-environment-broker-57fd5b957-hqpvj           1/1       Running            6          17h
kyma-system        core-sub-validator-5c64d45c8f-tzjgf                      2/2       Running            0          17h
kyma-system        core-ui-api-644b858cd6-ct4ww                             2/2       Running            0          17h
kyma-system        dex-678b6d868d-8d7vw                                     1/1       Running            0          17h
kyma-system        prometheus-core-0                                        2/2       Running            0          17h
kyma-system        prometheus-operator-57b98966c7-hk45l                     1/1       Running            0          17h

```

**Comment by abbi-gaurav** [comment]
@rkamath3 Thanks for reporting the issue.
We have witnessed this issue during our CIs and are working on fixing it. Please see #1406 

The underlying problem is the inaccessibility of dex due to `ingressgateway not receiving configuration from gateways`. 

Jaeger uses the `kcproxy` container as a side-car which tries to connect to dex and in case of connectivity issues with dex fails thus causing the pod to fail and restart.

**Comment by kubadz** [comment]
The issue should no longer occur in Kyma versions >0.4.3. As you can read in the issue #1406 we already merge the PR with a workaround to this problem. After that, no one reported a similar problem.
@abbi-gaurav I would say that this issue should be closed, not be "blocked" by the issue which is partially resolved (have a proper workaround to the problem, but is still open as we didn't find the problem cause).

**Comment by abbi-gaurav** [comment]
This issue has been fixed in #1373 PR.
